src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -68,10 +74,11 @@ ucs_config_field_t uct_ib_mlx5_iface_config_table[] = {
      "+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++\n",
      ucs_offsetof(uct_ib_mlx5_iface_config_t, ar_enable), UCS_CONFIG_TYPE_TERNARY_AUTO},
 
-    {"CQE_ZIPPING_ENABLE", "no",
+    {"CQE_ZIPPING_ENABLE", "",
```

Comment:
1. Maybe rename the param, as it is not a "flag" anymore. Consider how other `UCS_CONFIG_TYPE_BITMAP` params are named.
2. In any case, please make the explanation here more explicit, so that the user know exactly what are the possible values.

Reply:
1. What do you think about reverting the syntax? I mean introduce two `CQ_[TX/RX]_FEATURES` controllers. At this moment, the only possible values for them are "empty" or "cqe zipping". But if in the future we add controllable more features for CQ, it would be pretty extensible.

Reply:
Then it can be something like `CQ_[TX|RX]_MODE`. Not bad. @yosefe  WDYT?

Reply:
IMO, `CQ_[TX/RX]_FEATURES` is too generic.
We can have `[TX/RX]_CQ_ZIPPING_ENABLE`